package.json

```diff
@@ -42,6 +42,7 @@
     "acceptancetest:release": "nyc ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@release' --exit && ts-mocha packages/ws-server/tests/acceptance/index.spec.ts  -g '@release' --exit",
     "acceptancetest:release_http": "nyc ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@release' --exit",
     "acceptancetest:release_ws": "nyc ts-mocha packages/ws-server/tests/acceptance/index.spec.ts  -g '@release' --exit",
+    "acceptancetest:mainnet_light": "nyc ts-mocha packages/server/tests/acceptance/index.spec.ts  -g '@mainnet-light' --exit",
```

Comment:
what do you think of `release-light` instead of `mainnet-light` here, since I think we're likely to run this against all networks?

Reply:
Updated